packages/transport/src/transports/abstract.ts

```diff
@@ -93,7 +93,7 @@ export abstract class AbstractTransport extends TypedEmitter<{
     /**
      * once transport has been stopped, it does not emit any events
      */
-    protected stopped = false;
+    protected stopped = true;
```

Comment:
not sure if `stopped` is the best name, `inited` ? `running`?

Reply:
I would go for `dispose` and `disposed`